packages/strapi-plugin-users-permissions/services/RefreshToken.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+/**
+ * RefreshToken.js service
+ *
+ * @description: A set of functions similar to controller's actions to avoid code duplication.
+ */
+const randtoken = require('rand-token');
+
+module.exports = {
+  revoke: async params => {
+    params.model = 'refreshtoken';
+    return strapi.query('refreshToken', 'users-permissions').delete(params);
+  },
+
+  issue: async (user, useragent) => {
+    let token = randtoken.uid(255);
```

Comment:
Why you did not use this function https://github.com/strapi/strapi/blob/master/packages/strapi-plugin-users-permissions/services/Jwt.js#L41 to generate your token?

Reply:
I thought keeping their logic separate is much better if there will be any changes in any of them.